lib/luatest_server.py

```diff
@@ -34,10 +34,10 @@ def __init__(self, *args, **kwargs):
     def execute(self, server):
         """Execute test by luatest command
 
-        Execute `luatest -c --verbose <name>_test.lua --output tap` command.
-        Disable capture mode. Provide a verbose output in the tap format.
-        Extend the command by `--pattern <pattern>` if the corresponding
-        option is provided.
+        Execute `luatest -c --no-clean --verbose <name>_test.lua --output tap`
+        command. Disable capture mode and deletion of the var directory. Provide
```

Comment:
Please move word `Provide` to the next line.

Reply:
yep, fixed.